lib/ffmprb/filter.rb

```diff
@@ -242,6 +242,7 @@ def volume_exp(volume)
         prev_vol = volume[prev_at] || 1.0
         exp = "#{volume[volume.keys.last]}"
         volume.each do |at, vol|
+          next if at == prev_at
```

Comment:
well strictly speaking the "later" volume value should not be skipped, but where do you see this happening anyway?